spec/features/user_recievs_email_after_microhood_is_reviewed_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+feature 'User is notified via email after review of their created microhood' do
+  scenario 'user reviews microhood' do
+    ActionMailer::Base.deliveries = []
+    user = FactoryGirl.create(:user)
+    login_as(user, scope: :user)
+    microhood = FactoryGirl.create(:microhood)
+    visit microhood_path(microhood)
+    expect(page).to have_content microhood.street
```

Comment:
Don't have expectations in the middle of tests. If you need to have expectations at two places, you need to have two tests.